node/hello-world/index.js

```diff
@@ -39,18 +46,27 @@ const getRowGreeting = (row) => {
       key: `greeting${index}`,
       data: {
         [COLUMN_FAMILY_NAME]: {
-          [COLUMN_NAME]: greeting,
+          [COLUMN_NAME]: {
+            value: greeting,
+            timestamp: new Date(),
```

Comment:
Please add a comment saying that setting the date allows the client to perform retries.  If server-side time is used, retries may cause multiple cells to be generated.